Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py

```diff
@@ -244,11 +250,19 @@ def query_loggings(self, query: str) -> tuple[list[dict], list]:
                 error_message = query_result
 
             raise DemistoException(f'Error in query to Cortex Data Lake XSOAR Connector [{status_code}] - {error_message}')
-
+        raw_results = []
         try:
-            raw_results = [r.json() for r in query_service.iter_job_results(job_id=query_result.get('jobId'),
-                                                                            result_format='valuesDictionary',
-                                                                            max_wait=2000)]
+            for r in query_service.iter_job_results(job_id=query_result.get(
```

Comment:
Do we have any issues with page_number/page_size entered as None to the iter_job_results?

Reply:
<img width="523" alt="image" src="https://github.com/user-attachments/assets/cef11dba-516a-45db-ba2a-1301163fd58f">
Should be fine - the parameters are included internally only if they rnt None